src/Microsoft.Data.SqlClient/netcore/src/Resources/Strings.resx

```diff
@@ -931,7 +931,7 @@
     <value>Associating port with I/O completion mechanism failed</value>
   </data>
   <data name="SNI_ERROR_11" xml:space="preserve">
-    <value>Timeout error</value>
+    <value>Connection timed out.</value>
```

Comment:
You may change this in src\Microsoft.Data.SqlClient\netfx\src\Resources\Strings.resx to keep consistency.

Reply:
I didn't want to change NetFx though.. 🤔 
Might as well revert it for now.